src/api/java/com/minecolonies/api/tileentities/TileEntityRack.java

```diff
@@ -95,6 +95,12 @@ public int getFreeSlots()
         return freeSlots;
     }
 
+    @Override
+    public void tick()
+    {
+        combinedHandler = null;
```

Comment:
Nope, figure out a better way todo this.
Neighbor update or what ever.

But this is not a good reason to make this TileEntity ticking, it will cause a massive performance regression and that is not acceptable for something like this, since this adds no functionality what so ever to our tile.